File: haskell/iniciante/1035.hs
-- title: Teste de Seleção 1
-- number of the problem: 1035
-- date: 21/09/20
-- author: ltsdw

main :: IO ()
main = do
    l1 <- getLine

    let (a : b : c : d : _) = map (read :: String -> Int) $ words l1

    if (b > c) && (d > a) && ((c + d) > (a + b)) && (c > 0) && (d > 0) && even a
        then
            putStrLn "Valores aceitos"
    else
        putStrLn "Valores nao aceitos"

File: python/iniciante/1035.py
'''
title: Teste de Seleção 1
number of the problem: 1035
date: 02/03/2018
author: ltsdw
'''


a, b, c, d = map(int, input('').split())

if b > c and d > a and c + d > a + b and c > 0 and d > 0 and a % 2 == 0: print('Valores aceitos')
else: print('Valores nao aceitos')
